src/FluentConfiguration/PropertyConfiguration.cs

```diff
@@ -4,6 +4,15 @@ namespace FluentExcel
 {
     using System;
 
+    /// <summary>
+    /// Value validator delegate, validate value before <see cref="PropertyConfiguration.ValueConverter"/>
+    /// </summary>
+    /// <param name="rowIndex">Row index of current cell in excel</param>
+    /// <param name="columnIndex">Column index of current cell in excel</param>
+    /// <param name="value">value of current cell</param>
+    /// <returns>Wether the value passes validation</returns>
+    public delegate bool ValueValidatorDelegate(int rowIndex, int columnIndex, object value);
```

Comment:
According Microsoft's `Framework Design Guidelines`, we SHOULD first consider use `Func` or `Action` delegate.

Reply:
Sorry I don't know commits followed by pull request will become a part of the pull request. I thought the pull request will remains as it was until I update it. I didn't plan to feedback the subsequent code this soon. I was about to do some testing before another pull request, which is for this subsequent code, after the first pull request get merged.
And about the delegate thing, I prefer custom delegate much better to Func/Action when the delegate has multiple parameters, just what the current situation is. The reason is that compared with Func/Action, custom delegate is more descriptive. Each parameter of the custom delegate can get a name. So that the people who is using it can get a idea about what each parameter is for immediately.

Reply:
So as a sum-up, the code are not ready for merging right now. And sorry for the disturbance, I will let you know when it's ready to merge.
Currently I referenced the FluentExcel project in some projects and is about to write some code using it. After this is done and the code get tested, I'll notify you here.

Reply:
OK, I see